src/govuk/components/character-count/character-count.unit.test.mjs

```diff
@@ -0,0 +1,148 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { createElement } from '../../../../lib/dom-helpers.mjs'
+import CharacterCount from './character-count.mjs'
+
+describe('CharacterCount', () => {
+  describe('formatCountMessage', () => {
+    describe('default configuration', () => {
+      let component
+      beforeAll(() => {
+        // The component won't initialise if we don't pass it an element
+        component = new CharacterCount(createElement('div'))
+      })
+
+      it('formats singular remaining characters', () => {
+        expect(component.formatCountMessage(1, 'characters')).toEqual('You have 1 character remaining')
+      })
+      it('formats plural remaining characters', () => {
+        expect(component.formatCountMessage(10, 'characters')).toEqual('You have 10 characters remaining')
+      })
+      it('formats singular exceeding characters', () => {
+        expect(component.formatCountMessage(-1, 'characters')).toEqual('You have 1 character too many')
+      })
+      it('formats plural exceeding characters', () => {
+        expect(component.formatCountMessage(-10, 'characters')).toEqual('You have 10 characters too many')
+      })
+      it('formats character limit being met', () => {
+        expect(component.formatCountMessage(0, 'characters')).toEqual('You have no characters remaining')
+      })
+
+      it('formats singular remaining words', () => {
+        expect(component.formatCountMessage(1, 'words')).toEqual('You have 1 word remaining')
+      })
+      it('formats plural remaining words', () => {
+        expect(component.formatCountMessage(10, 'words')).toEqual('You have 10 words remaining')
+      })
+      it('formats singular exceeding words', () => {
+        expect(component.formatCountMessage(-1, 'words')).toEqual('You have 1 word too many')
+      })
+      it('formats plural exceeding words', () => {
+        expect(component.formatCountMessage(-10, 'words')).toEqual('You have 10 words too many')
+      })
+      it('formats word limit being met', () => {
+        expect(component.formatCountMessage(0, 'words')).toEqual('You have no words remaining')
+      })
+
+      it('formats the count', () => {
```

Comment:
We're using the word format in the test descriptions to mean a few different things – can we be clearer about what it is we're testing?